nrf_wifi/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -88,7 +70,259 @@
 #define NRF_WIFI_VBAT_LOW  (12)  /* Correspond to (2.5+12*0.07)=3.34V */
 #define NRF_WIFI_VBAT_HIGH (14) /* Correspond to (2.5+14*0.07)=3.48V */
 
+/* Package independent params */
+
+/**  Power detector adjustment value.*/
```

Comment:
```suggestion
/** Power detector adjustment value. */
```
This and for all others.